src/web/common.rkt

```diff
@@ -86,55 +86,55 @@
 
 (define (preprocess->c preprocess)
   (match preprocess
-    [(list 'sort variables ...)
+    [(list 'sort variables)
```

Comment:
Did you change the format here? Otherwise, this seems wrong.

Reply:
Yeah, I changed it to just be a pair of the type and associated data